README.md

```diff
@@ -1,175 +1,4 @@
-# General Grants Program <!-- omit in toc -->
+# General Grants Program
 
-<p align="center">
-  <img src="src/General_Grants_Program.png" style="width:1300px";>
-</p>
+This repository has been deprecated in favour of the [Open Grants repository](https://github.com/w3f/Open-Grants-Program/). You can still apply for a General Grant - to do so, follow the steps outlined in Open Grant Program's README.
```

Comment:
Perhaps we could even remove this distinction? We'd simply have a W3F Grants Program and for applications that that are private, above 100k and all that - a link to the form. I think David initially suggested this.

Reply:
I don't strictly favor one over the other, but I think it'd help to have a separate name for it, just so we can point people to the right resources.

Reply:
I would be in favor of just calling it Web3 Grants.

Reply:
Maybe we could add an explanation here that now it’s only Web3 Grants and that we merged the old open grants + general grants into one repo.

Reply:
```suggestion
This repository has been deprecated in favour of the [W3F Grants repository](https://github.com/w3f/Grants-Program/).
```